ray-operator/config/samples/ray_v1alpha1_rayservice.yaml

```diff
@@ -109,3 +109,6 @@ spec:
                   requests:
                     cpu: "500m"
                     memory: "2Gi"
+    serviceAnnotations:
+      # annotations for the Service
+      service_key: "service_value"
```

Comment:
How about using comments to describe the usage and leaving the annotations empty `{}`?